std/file.d

```diff
@@ -245,6 +233,17 @@ void[] read(R)(R name, size_t upTo = size_t.max)
         return readImpl(null, name.tempCString!FSChar(), upTo);
 }
 
+///
+@safe unittest
+{
+    import std.utf : byChar;
+    scope(exit) { assert(exists(deleteme)); remove(deleteme); }
```

Comment:
I'm not sure `deleteme` is a good name to use as an example in the documentation.